### ISSUE TITLE
View custom PDB structures

### ISSUE THREAD

**Issue by sacdallago** [ebi-webcomponents/nightingale#73]
**Is your feature request related to a problem? Please describe.**
I cannot seem to display structures from predictions (e.g. loading external resources). Usually I (and many I know) have several models for the same protein sequence, so the view as on UniProt (with a table/list on the side to select from a list of structures) is very useful and could be reused 1-1.

**Describe the solution you'd like**
An attribute in `protvista-structure` for loading data from external resources, or better: a data loader.

**Describe alternatives you've considered**
Rewriting litemol to load PDB structures from a URL

**Comment by xwatkins** [comment]
Good idea - the only difficulty is that we make 2 calls to the PDBe API, one to load the structure binary (which is the one you're talking about), the other to get the residue mappings (`https://www.ebi.ac.uk/pdbe/api/mappings/uniprot/${pdbId}`) which allow us to know which residue to highlight in the structure when a user selects a feature in a track. Not sure how we could handle this with external resources?

**Comment by sacdallago** [comment]
Very good point. For structure prediction purposes, a mapping is usually not necessary (this can be stated as an "intended limitation"). The expectation is that the numbering in the PDB binary is already mapped to the correct sequence numbering.

TL;DR: keep it simple, expect the numbering in the PDB file to match the sequence.

**Comment by xwatkins** [comment]
👍
